### PR TITLE
Disable a permared suite.

### PR DESCRIPTION
**Pull request by tvalentyn**
related: #32492  #30519 

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by tvalentyn** [comment]
R: @damccorm @abacn @lostuck

File: .github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml [hunk 1]
```diff
@@ -60,7 +61,7 @@ jobs:
     name: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
     strategy:
       matrix:
-        job_name: ["beam_PostCommit_XVR_GoUsingJava_Dataflow"]
+        job_name: ["beam_PostCommit_XVR_GoUsingJava_Dataflow(DISABLED)"]
```
Comment (tvalentyn) [inline .github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml:64]: LMK if there is a smarter way to disable a suite.

File: .github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml [hunk 2]
```diff
@@ -84,8 +85,9 @@ jobs:
           USER: github-actions
           CLOUDSDK_CONFIG: ${{ env.KUBELET_GCLOUD_CONFIG_PATH}}
         uses: ./.github/actions/gradle-command-self-hosted-action
-        with:
-          gradle-command: :runners:google-cloud-dataflow-java:validatesCrossLanguageRunnerGoUsingJava
+        # TODO(https://github.com/apache/beam/issues/32492): re-enable the suite.
+        # with:
+        #   gradle-command: :runners:google-cloud-dataflow-java:validatesCrossLanguageRunnerGoUsingJava
```
Comment (Abacn) [inline .github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml:90]: I think we can preserve this "with: gradle-command". We only need to comment out the "schedule" trigger (as you did) and pull_request_target trigger (which prevents this test suite run on release branch verification)

File: .github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml (hunk 1 above)
Comment (Abacn) [reply .github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml:64]: thanks for taking care of this, commented below.

File: .github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml (hunk 2 above)
Comment (tvalentyn) [reply .github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml]: Ah, great, that's better. thanks!

Comment (tvalentyn) [reply .github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml]: Actually, we don't allow this:

* What went wrong:
Execution failed for task ':beam-test-gha:check'.
> Check failed: Fail to get the trigger path for beam_PostCommit_XVR_GoUsingJava_Dataflow.yml. Make sure it has a pull_request_target trigger.

Comment (tvalentyn) [reply .github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml]: I suppose I could write in /dev/null to silence it

Comment (tvalentyn) [reply .github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml]: would that work with the release script?

Comment (tvalentyn) [reply .github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml]: nvm, i'll just remove the trigger all tests.json

Comment (tvalentyn) [reply .github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml]: PTAL

Comment (damccorm) [reply .github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml]: Lol, the script doesn't like that either. We probably should just insert some logic in https://github.com/apache/beam/blob/eb8639b013e9d820a253970ed03ac0e7cfc6e13e/.github/build.gradle#L41 to allow disabling.

Maybe we could just have a string check that we have something like `gha-check-ignore` defined somewhere in the file contents? Or we could have a separate file like `release/trigger_all_disabled_workflows.json` and check that you're either triggering on that OR on `release/trigger_all_tests.json`?

**Review by Abacn — APPROVED** [review]

File: .github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml (hunk 2 above)
Comment (Abacn) [reply .github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml]: ah, this was a guard to make sure newly added tests are exercised in release branch verification, yeah we should have an exclusion list

Comment (tvalentyn) [reply .github/workflows/beam_PostCommit_XVR_GoUsingJava_Dataflow.yml]: we can analyze workflow name to filter out disabled workflows.

**Review by damccorm — APPROVED** [review]
Thanks, LGTM

**Pull request merged**